cmd/boltzcli/commands.go

```diff
@@ -1540,6 +1540,16 @@ var walletCommands = &cli.Command{
 			Action:    requireNArgs(1, listTransactions),
 			Flags:     []cli.Flag{jsonFlag, &cli.BoolFlag{Name: "exclude-swap-related", Usage: "Exclude swap related transactions"}},
 		},
+		{
+			Name:   "bumpfee",
+			Usage:  "Bump the fee of a transaction",
+			Action: bumpFee,
+			Flags: []cli.Flag{
+				&cli.StringFlag{Name: "swap-id", Usage: "Swap which should be used to get the transaction"},
+				&cli.StringFlag{Name: "tx-id", Usage: "ID of the transaction to bump. Has to belong to a clients wallet."},
+				&cli.Float64Flag{Name: "fee-rate", Usage: "Fee rate in sat/vbyte"},
```

Comment:
https://github.com/BoltzExchange/boltz-client/pull/388/files/5d6311620787302ec857d51d23bddd69fcd6564c#r2005904220 also in the CLI